utilities/xmlrpcpp/src/XmlRpcClient.cpp

```diff
@@ -312,6 +312,11 @@ XmlRpcClient::generateRequest(const char* methodName, XmlRpcValue const& params)
                   header.length(), body.length());
 
   _request = header + body;
+  // Limit the size of the request to avoid integer overruns
+  if (_request.length() > __INT_MAX__) {
+    XmlRpcUtil::error("XmlRpcClient::generateRequest: oversize message length.");
```

Comment:
For debugging, it would be helpful to print the max size as well as actual size here. This applies below as well.

Reply:
Addressed in 2404bd7fb0c2fb60c1950bf7101b200723660a0a